tests/keras/test_callback.py

```diff
@@ -111,3 +111,7 @@ def test_keras_mlflow_callback_log_every_n_steps():
         key="validation_loss",
     )
     assert len(validation_loss_history) == num_epochs
+
+
+def test_old_callback_still_exists():
+    assert mlflow.keras.MLflowCallback is mlflow.keras.MlflowCallback
```

Comment:
```suggestion
    assert mlflow.keras.MLflowCallback is mlflow.keras.MlflowCallback

```

Reply:
@B-Step62 There a new line at the end of the file. If there wasn't, ruff would complain.

Reply:
ooh so it's just github doesn't show the empty line at the end, sorry!